lib/fetch/response.js

```diff
@@ -512,6 +487,23 @@ function initializeResponse (response, init, body) {
   }
 }
 
+/**
+ * @param {any} innerResponse
```

Comment:
Can you link to https://fetch.spec.whatwg.org/#response-create?

Reply:
```suggestion
 * @see https://fetch.spec.whatwg.org/#response-create
 * @param {any} innerResponse
```